test/unit-test/shadow_utest.c

```diff
@@ -398,6 +663,121 @@ void test_Shadow_GetTopicString_Happy_Path( void )
         shadowStatus = Shadow_GetTopicString( topicTypes[ index ],
                                               TEST_THING_NAME,
                                               TEST_THING_NAME_LENGTH,
+                                              SHADOW_NAME_CLASSIC,
+                                              0,
+                                              &( topicBuffer[ 0 ] ),
+                                              bufferSize,
+                                              &outLength );
+        TEST_ASSERT_EQUAL_INT( expectedBuffersSize[ index ], outLength );
+        TEST_ASSERT_EQUAL_STRING_LEN( expectedBuffers[ index ],
+                                      topicBuffer,
+                                      outLength );
+    }
+}
+
+/**
+ * @brief Tests the behavior of Shadow_GetTopicString() for named shadows, with valid parameters.
+ */
+void test_Shadow_Named_GetTopicString_Happy_Path( void )
+{
+    ShadowStatus_t shadowStatus = SHADOW_SUCCESS;
+    uint16_t outLength = 0;
```

Comment:
```suggestion
    uint16_t outLength = 0U;
```

Reply:
The existing unit tests have lots of zero initializers like this. So I did it the same way. I just presumed MISRA did not apply on the test code? I'm happy to change all of them to 0U.

Reply:
MISRA checks are not done for test code.  However, it will be better to keep the initialized values of the same type as that of the type of the variable. It will be better if change them to 0U for the code you are bringing in. Thanks for pointing out the inconsistency that will create with the other parts of the code. Yea. We will do a follow up PR and fix it.

Reply:
Done.